FLEDGE.md

```diff
@@ -231,9 +231,9 @@ Buyers may want to make on-device decisions that take into account real-time dat
 
     https://www.kv-server.example/getvalues?hostname=publisher.com&keys=key1,key2
 
-The response from the server should be a JSON object whose keys are key1, key2, etc., and whose values will be made available to the buyer's bidding functions.
+The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trusted_bidding_signals_url`, the hostname of the top-level webpage `publisher.com` is provided by the browser, and `keys` is a list of `trusted_bidding_signals_keys` strings, perhaps coalesced (for efficiency) across any number of interest groups that share a `trusted_bidding_signals_url`.  The response from the server should be a JSON object whose keys are key1, key2, etc., and whose values will be made available to the buyer's bidding functions (un-coalesced).
```

Comment:
If the hostname is part of the query string, it has to match, in addition to the trusted_bidding_signals_url.  Alternatively, remove the hostname parameter (assuming it's embedded in the URL itself), and allow merging across hostnames.

Reply:
Oh, and this is much clearer, thanks!

Reply:
Ah sorry!  The hostname is the _publisher_ site, i.e. the page the winning ad is going to be displayed on.  It has nothing to do with the owner of the interest group.